docs/articles/expensify-classic/bank-accounts-and-credit-cards/company-cards/Direct-Bank-Connections.md

```diff
@@ -1,5 +1,108 @@
 ---
 title: Direct Bank Connections
-description: Direct Bank Connections
+description: Connect your company cards in Expensify to bring all team members’ card expenses into their accounts and conveniently manage card transactions and out-of-pocket expenses in one place.
+
 ---
-## Resource Coming Soon!
+# Overview
+If you're a Domain Admin, you have the power to connect and manage your company cards in Expensify centrally. If your company uses a card program with one of our Approved! Banking Partners, you can easily connect the card feed to Expensify via login credentials. Connecting company cards is a great way to bring all team members’ card expenses into their accounts and conveniently manage card transactions and out-of-pocket expenses in one place. Keeping things organized has never been easier!
+# How to connect company cards using a direct bank connection
+1. Go to **Settings > Domains > _Domain Name_ > Company Cards**
+2. Click **Import Card** 
+
+![Insert alt text for accessibility here](https://help.expensify.com/assets/images/ExpensifyHelp_DomainCards.png){:width="100%"}
```

Comment:
Is it normal it looks like that? (with the width)
<img width="704" alt="image" src="https://github.com/Expensify/App/assets/2463975/845990dc-0550-4903-97f7-fcb4d9e86e0b">

And should you insert some alt text?

(same for all the images below)